ctapipe/utils/datasets.py

```diff
@@ -31,19 +31,21 @@
 DEFAULT_URL = "http://cccta-dataserver.in2p3.fr/data/ctapipe-extra/v0.3.3/"
 
 
-def get_searchpath_dirs(searchpath=os.getenv("CTAPIPE_SVC_PATH")):
+def get_searchpath_dirs(searchpath=os.getenv("CTAPIPE_SVC_PATH"), url=DEFAULT_URL):
     """ returns a list of dirs in specified searchpath"""
+
+    print(searchpath)
     if searchpath == "" or searchpath is None:
         searchpaths = []
     else:
         searchpaths = [Path(p) for p in os.path.expandvars(searchpath).split(":")]
 
-    searchpaths.append(get_cache_path(DEFAULT_URL))
+    searchpaths.append(get_cache_path(url))
 
     return searchpaths
 
 
-def find_all_matching_datasets(pattern, searchpath=None, regexp_group=None):
+def find_all_matching_datasets(pattern, url=DEFAULT_URL, searchpath=None, regexp_group=None):
```

Comment:
Better add new arguments at the end, so you don't break API (since now the order of args has changed)

Reply:
done